src/audio/host-zephyr.c

```diff
@@ -403,9 +406,16 @@ static uint32_t host_get_copy_bytes_normal(struct host_data *hd, struct comp_dev
 	if (!(hd->ipc_host.feature_mask & BIT(IPC4_COPIER_FAST_MODE)))
 		dma_copy_bytes = MIN(hd->period_bytes, dma_copy_bytes);
 
-	if (!dma_copy_bytes)
-		comp_info(dev, "no bytes to copy, available samples: %d, free_samples: %d",
-			  avail_samples, free_samples);
+	if (!dma_copy_bytes) {
+		static uint64_t nobytes_last_logged;
+		uint64_t now = k_uptime_get();
+
+		if (now - nobytes_last_logged > SOF_MIN_NO_BYTES_INTERVAL_MS) {
+			nobytes_last_logged = now;
+			comp_info(dev, "no bytes to copy, available samples: %d, free_samples: %d",
+				  avail_samples, free_samples);
```

Comment:
Idea: add a counter. Log how many messages have been hidden like
"no bytes to copy, %u such events in last %u ms, available samples: %d, free_samples: %d",